internal/test/wmcb/main_test.go

```diff
@@ -17,7 +18,14 @@ var (
 )
 
 func TestMain(m *testing.M) {
-	err := framework.Setup(vmCount)
+	var vmCreds e2ef.Creds
+	var skipVMSetup bool
+
+	flag.Var(&vmCreds, "vmCreds", "List of VM credentials")
```

Comment:
Shouldn't we expose this option as a string and then use https://github.com/openshift/windows-machine-config-operator/pull/122/files#diff-5465ccc52b3a24461a98f029910cf60aR42 method to get the slice?

Reply:
That is another way of doing it. I chose to go with this approach.